packages/react-core/src/components/EmptyState/examples/EmptyStateBasic.tsx

```diff
@@ -1,32 +1,33 @@
 import React from 'react';
 import {
-  Title,
   Button,
   EmptyState,
-  EmptyStateIcon,
   EmptyStateBody,
-  EmptyStateSecondaryActions
+  EmptyStateActions,
+  EmptyStateHeader,
+  EmptyStateFooter
 } from '@patternfly/react-core';
 import CubesIcon from '@patternfly/react-icons/dist/esm/icons/cubes-icon';
 
 export const EmptyStateBasic: React.FunctionComponent = () => (
   <EmptyState>
-    <EmptyStateIcon icon={CubesIcon} />
-    <Title headingLevel="h4" size="lg">
-      Empty state
-    </Title>
+    <EmptyStateHeader titleText="Empty state" icon={CubesIcon} />
```

Comment:
For all of the Empty State examples, let's pass in `headingLevel="h4"` to EmptyStateHeader. Right now Axe flags subsequent example headings since the EmptyStateHeader is using an h1, so it looks like we're going from an h1 to an h3 heading.